crates/forge/src/lib.rs

```diff
@@ -101,6 +102,7 @@ enum ForgeSubcommand {
         args: CleanArgs,
     },
     /// Clean Forge cache directory
+    #[command(name = "clean-cache", hide = true)]
```

Comment:
Don't think we need to override the default name

```suggestion
    #[command(hide = true)]
```